packages/core/src/indexing-store/utils/errors.ts

```diff
@@ -1,4 +1,5 @@
 import {
+  ConstraintCheckError,
```

Comment:
Should be `CheckConstraintError`

https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-CHECK-CONSTRAINTS